Update EEx identifiers to match vscode-elixir-ls

aj-foster opened the pull request:
The [VS Code ElixirLS extension](https://github.com/JakeBecker/vscode-elixir-ls/) recently updated to use `EEx` and `HTML (EEx)` (with a capitalized second `E`) in its grammar identifiers. As a result, the snippets contained in this package became unavailable in EEx files. This change updates the snippet contribution to match the new identifiers in that package.

Note that, if this change is accepted, users of the [no-longer-maintained](https://github.com/fr1zle/vscode-elixir/issues/128#issuecomment-478668010) [VS Code Elixir](https://github.com/fr1zle/vscode-elixir/) extension will face the same issue, in reverse. Relevant communication might be advisable.

See https://github.com/JakeBecker/vscode-elixir-ls/issues/134 for more information.

aj-foster committed:
Update EEx identifiers to match vscode-elixir-ls

The VS Code ElixirLS extension, https://github.com/JakeBecker/vscode-elixir-ls/,
recently updated to use `EEx` and `HTML (EEx)` (with a capitalized second `E`)
in its grammar identifiers. As a result, the snippets contained in this package became
unavailable in EEx files. This change updates the snippet contribution to match the
new identifiers in that package.

See https://github.com/JakeBecker/vscode-elixir-ls/issues/134 for more information.

florinpatrascu commented:
any updates on this? Otherwise we have to edit the `~/.vscode/extensions/stefanjarina.vscode-eex-snippets-0.0.3/package.json` manually.

stefanjarina commented:
Hi, looking into this, there is definitely way to support both extensions "ElixirLS" and "vscode-elixir"

aj-foster committed:
Support previous lowercase `Eex` configuration

This change should allow users with the other popular Elixir language extension for VS Code
to continue using snippets.

aj-foster commented:
You are correct; I believe the necessary changes are now on this branch.

florinpatrascu commented:
Thank you!

stefanjarina commented:
Thank you as well, for pointing out this issue and for pull request.
Also bumped version to 0.0.4 and publish to marketplace updated version so you should get update in VSCode soon.
